### ISSUE TITLE
Meeting with Noe

### ISSUE THREAD

**Issue by ilyaperepelitsa** [kaleidopop/Development_1#3]
Did everyone get the email?
I won't be able to attend the one next Tuesday, I have classes from 6 to 10. We can either have someone meet him on Tuesday or tomorrow. I have nothing planned for tomorrow so I can try to sign us up. Let me know if that works. If you can come please post the hours that you're free. I think we should aim at maximizing the number of people who can attend.

**Comment by kaleidopop** [comment]
Dear all,

I am meeting Noe at 1:30pm on Friday. I am happy to meet him with you all
around 2pm or else at least we should all meet once together before we meet
him.

Let me know how best to proceed. My # is 917-930-4552, we can also text to
coordinate a day and time to get this project started and to schedule time
to meet with Noe.

d.

**Comment by lynek169** [comment]
Hey guys,

Hate to be difficult but I'm out of town right now for a wedding. I also have classes next Tuesday from 4-8pm. I suggested to Noe having you guys meet with him and I'd get an update from you guys (just so we're not delayed in starting) but if we want to wait until all 4 of us are available that's an option too. 

Kait

**Comment by ilyaperepelitsa** [comment]
Lets meet around 2 and figure out wether we need to meet him tomorrow or have someone come to his office hours on tuesday. I'm trying to make sense out of that college dataset and it's been quite tough to navigate and manipulate it. I'll try my best to come up with at least a data frame with historical average SAT scores for tomorrow - if I manage to do that - it's 80% of success with this database. 
Where are you meeting him? can come there so that you don't have to run and look for people

**Comment by kaleidopop** [comment]
I am meeting him in Econ dept in the same building where we have class. Is
there a cafe around there or a public space? Or just come to Econ dept
around 2. If we discuss our data set and questions we can just tell him
right then. Otherwise I am able to go to his office hours on Tuesday.

**Comment by ilyaperepelitsa** [comment]
I'll be there at 2! Gonna be waiting in the cafe feel free to text me - 8572107709

**Comment by saharalhassan** [comment]
Hey Guys! I'm sorry I won't be able to meet Noe with you today. I have a paper due later today and I'm not done yet. I'll try to see him later this week.

**Comment by kaleidopop** [comment]
Ilya, I actually have to cancel my meeting today. I have a paper due too
and I am rushing to finish it. I am going to cancel with Noe as well. Can
we try to coordinate on the phone this weekend? Group call?
d.

**Comment by ilyaperepelitsa** [comment]
We could just meet after the midterm on Monday. Once we figure the things out on Monday we could send people to his office hours on Tuesday. 
Let me know what you guys think. I am trying to learn how to do these things with R that can get toxic for group performance so we just need to figure out what we're doing specifically, who's responsible for things and so forth.

**Comment by lynek169** [comment]
Yeah, I definitely think it will be helpful to meet quickly after class on
Monday. Good luck on the papers!

On Fri, Nov 6, 2015 at 12:15 PM, ilyaperepelitsa notifications@github.com
wrote:

> We could just meet after the midterm on Monday. Once we figure the things
> out on Monday we could send people to his office hours on Tuesday.
> Let me know what you guys think. I am trying to learn how to do these
> things with R that can get toxic for group performance so we just need to
> figure out what we're doing specifically, who's responsible for things and
> so forth.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kaleidopop/Development_1/issues/3#issuecomment-154490187
> .

## 

_KAITLYN LYNES_
_PROJECT ASSISTANT_
JULIAN J. STUDLEY GRADUATE PROGRAM IN INTERNATIONAL AFFAIRS
_MILANO SCHOOL OF INTERNATIONAL AFFAIRS, MANAGEMENT, AND URBAN POLICY_

72 FIFTH AVE, NEW YORK, NY 10011
lynek169@newschool.edu
T 212.229.5400 x1606 / M 815.355.1141

@kait_lynes https://twitter.com/kait_lynes
newschool.edu/milano
[image: The New School]

**Comment by lynek169** [comment]
Hey guys,

I'm not exactly sure where to look to check out the data sets. Can anyone
lead me to the correct place?

Thanks,
Kait

On Fri, Nov 6, 2015 at 1:46 PM, Kaitlyn Lynes lynek169@newschool.edu
wrote:

> Yeah, I definitely think it will be helpful to meet quickly after class on
> Monday. Good luck on the papers!
> 
> On Fri, Nov 6, 2015 at 12:15 PM, ilyaperepelitsa <notifications@github.com
> 
> > wrote:
> > 
> > We could just meet after the midterm on Monday. Once we figure the things
> > out on Monday we could send people to his office hours on Tuesday.
> > Let me know what you guys think. I am trying to learn how to do these
> > things with R that can get toxic for group performance so we just need to
> > figure out what we're doing specifically, who's responsible for things and
> > so forth.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/kaleidopop/Development_1/issues/3#issuecomment-154490187
> > .
> 
> ## 
> 
> _KAITLYN LYNES_
> _PROJECT ASSISTANT_
> JULIAN J. STUDLEY GRADUATE PROGRAM IN INTERNATIONAL AFFAIRS
> _MILANO SCHOOL OF INTERNATIONAL AFFAIRS, MANAGEMENT, AND URBAN POLICY_
> 
> 72 FIFTH AVE, NEW YORK, NY 10011
> lynek169@newschool.edu
> T 212.229.5400 x1606 / M 815.355.1141
> 
> @kait_lynes https://twitter.com/kait_lynes
> newschool.edu/milano
> [image: The New School]

## 

_KAITLYN LYNES_
_PROJECT ASSISTANT_
JULIAN J. STUDLEY GRADUATE PROGRAM IN INTERNATIONAL AFFAIRS
_MILANO SCHOOL OF INTERNATIONAL AFFAIRS, MANAGEMENT, AND URBAN POLICY_

72 FIFTH AVE, NEW YORK, NY 10011
lynek169@newschool.edu
T 212.229.5400 x1606 / M 815.355.1141

@kait_lynes https://twitter.com/kait_lynes
newschool.edu/milano
[image: The New School]

**Comment by ilyaperepelitsa** [comment]
https://github.com/kaleidopop/Development_1/issues/2

**Comment by ilyaperepelitsa** [comment]
click the thingy

**Comment by lynek169** [comment]
Thanks! So from what I briefly see I like the idea mentioned earlier.
Comparing three countries, the US, and then one country ranked above in
terms of amount of grants given and one ranked below. Possible variables we
could use might include degree type, whether the school is public, private
nonprofit, or private for-profit, and program completion rate. I'm
literally just throwing out variables that sound interesting for me, but we
could do anything, obviously.

I told Noe we'd like to meet with him before class on Monday and said he
was free. We can meet with him in the same room he holds office hours 1126.
Does 7pm sound good for everyone? He also said it would be helpful if we
come with a research question and variables we'd like to use, so we should
definitely nail that down before Monday.

Looking forward to figuring out our research question!

Kait

On Thu, Nov 12, 2015 at 4:41 PM, ilyaperepelitsa notifications@github.com
wrote:

> click the thingy
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kaleidopop/Development_1/issues/3#issuecomment-156244779
> .

## 

_KAITLYN LYNES_
_PROJECT ASSISTANT_
JULIAN J. STUDLEY GRADUATE PROGRAM IN INTERNATIONAL AFFAIRS
_MILANO SCHOOL OF INTERNATIONAL AFFAIRS, MANAGEMENT, AND URBAN POLICY_

72 FIFTH AVE, NEW YORK, NY 10011
lynek169@newschool.edu
T 212.229.5400 x1606 / M 815.355.1141

@kait_lynes https://twitter.com/kait_lynes
newschool.edu/milano
[image: The New School]

**Comment by ilyaperepelitsa** [comment]
yeah I emailed him recently with the general idea of what we want to work on, he said it sounds interesting. Please lets not bullshit on it, let's put some real work. Wanna meet on Monday? like afternoon? I'll post a doodle here right now

**Comment by ilyaperepelitsa** [comment]
http://doodle.com/poll/ehwc73hdcmk58hym

**Comment by lynek169** [comment]
Thanks, can you let us know what you told him our idea was? I didn't get the impression we had decided as a group. I also understand you want to do a cool, indepth analysis, but working with four varying degrees of comfort with statistics and R, we shouldn't make this more complicated than it needs to be or else you'll end up doing most of the work.

**Comment by ilyaperepelitsa** [comment]
Don't worry, I want it to be group work too. I told him that we have several things that we want to work with including college data and NSF grants data and if we choose the level of difficulty he expects from us we might just do the NSF grants and figure out things like colleges that have unusually high frequency of receiving grants, specific professor, states, maybe map duration of research and so forth. 
The problem with any data set that includes historical data is that we will have to group things - It is gonna be easier for NSF grants because there are methods in the book that do just that but generally I would say in any data set we will have to do manipulations with data and that stuff is hard. That is what making certain things difficult (connecting two data frames) but this particular thing is optional
Let's start working with NSF thing on Monday so that at least we have something started

**Comment by kaleidopop** [comment]
Filled out the doodle. See you this afternoon.

**Comment by ilyaperepelitsa** [comment]
If we meet at 5 and have Kaitlyn joining at 6 do you think we'll have enough time to get things done?

**Comment by kaleidopop** [comment]
I am ok with that!!!

**Comment by lynek169** [comment]
Let me know where to meet you guys

**Comment by ilyaperepelitsa** [comment]
Let's meet on the 6th floor, there should be plenty of room there. I will bring the datasets (NSF stuff became very difficult to download all of a sudden)

**Comment by ilyaperepelitsa** [comment]
anyone close? I'm sitting at the desk in the far right corner of 6th floor

**Comment by kaleidopop** [comment]
I am on the train heading down...
